include/seastar/http/file_handler.hh

```diff
@@ -51,7 +51,8 @@ public:
      */
     virtual output_stream<char> transform(std::unique_ptr<http::request> req,
             const sstring& extension, output_stream<char>&& s) = 0;
-
+    file_transformer() = default;
+    file_transformer(const file_transformer&) = default;
```

Comment:
AFAIU, file_transformers are sugar-objects to push into route's handler. The way routes maintain handlers is weird by itself, but it seems to me that handlers are non-copyable too. Maybe move-able

Reply:
thank you for pointing this out! yeah, de-copyablized and moveablized.